.github/workflows/createWebNews.yaml

```diff
@@ -11,6 +11,9 @@ on:
             - Official
           default: "Beta"
 
+  schedule:
```

Comment:
Schedule is OK, but how do we choose if it's BETA or LIVE?

A better approach would be to release BETA @10 for example, followed by LIVE news release @11.

The release @11 can be done if spreadsheet text for that date has a specific text (confirmed/released whatever).

Reply:
Logic now is that we check if status column has "Released" in it as by 10AM all releases should be done already. If it has it, then we proceed with Web Post to Beta, if it doesn't - we notify SDK channel to check the spreadsheet/calendar.